table/options.go

```diff
@@ -336,6 +336,16 @@ func (m Model) WithFilterInputValue(value string) Model {
 	return m
 }
 
+// WithFilterFunc adds a filter function to the model. If the function returns a
+// truthy value, the row will be included in the filtered results. If the function
```

Comment:
`truthy value` is bleeding a bit of Javascript :D

Just `If the function returns true` for clarity.

Reply:
:facepalm: I just wrapped up writing a bunch of JS for some CI/CD pipelines. I guess my brain hadn't fully switched to Go-mode.